android/app/src/main/java/org/fossasia/openevent/viewmodels/AboutFragmentViewModel.java

```diff
@@ -28,34 +28,33 @@
 
     public AboutFragmentViewModel() {
         realmRepo = RealmDataRepository.getDefaultInstance();
-        sessions = new MutableLiveData<>();
-        dateList = new MutableLiveData<>();
-        eventLiveData = new MutableLiveData<>();
-        subscribeToEvent();
         subscribeToDateList();
     }
 
-    private void subscribeToSessions(ArrayList<String> dates) {
-        LiveRealmData<Session> sessionLiveRealmData = RealmDataRepository.asLiveData(realmRepo.getBookMarkedSessions());
-        sessions = Transformations.map(sessionLiveRealmData, input -> getSessionsList(dates, input));
-    }
-
     private void subscribeToDateList() {
-        RealmResults<EventDates> eventDates = realmRepo.getEventDatesSync();
-        ArrayList<String> dateListString = new ArrayList<>();
-        for (EventDates eventDate : eventDates) {
-            dateListString.add(eventDate.getDate());
+        if (dateList == null || sessions == null) {
```

Comment:
We should not need the subscribe to date list meth9d

Reply:
@iamareebjamal  subscribeToDateList method removed. Please review